sdks/python/setup.cfg

```diff
@@ -25,4 +25,5 @@ verbosity=2
 # fast_coders module which is not available when running unit tests:
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
-
+# Include end-to-end tests.
+include=wordcount_it
```

Comment:
`wordcount_it` is not included by nose default rules. I guess the reason is that the name is not end up with `_it`. Since in Java, it's called `WordCountIT`. Just want to have similar one. Probable `wordcount_it_test` will be included by default.